tests/test_utils/__init__.py

```diff
@@ -20,3 +20,11 @@
 AIRFLOW_MAIN_FOLDER = os.path.realpath(
     os.path.join(os.path.dirname(os.path.realpath(__file__)), os.pardir, os.pardir)
 )
+
+EXAMPLE_DAGS_FOLDER = os.path.join(
+    AIRFLOW_MAIN_FOLDER, "airflow", "example_dags"
+)
+
+TEST_DAGS_FOLDER = os.path.realpath(
```

Comment:
This is already set from airflow.settings.TEST_DAGS_FOLDER - we don't need another constant